packages/@aws-cdk/aws-glue-alpha/lib/jobs/pysparkflex-etl-job.ts

```diff
@@ -0,0 +1,180 @@
+import * as iam from 'aws-cdk-lib/aws-iam';
+import { Bucket } from 'aws-cdk-lib/aws-s3';
+import { CfnJob } from 'aws-cdk-lib/aws-glue';
+import { Job, JobProperties } from './job';
+import { Construct } from 'constructs';
+import { JobType, GlueVersion, JobLanguage, PythonVersion, WorkerType, ExecutionClass } from '../constants';
+import { SparkUIProps, SparkUILoggingLocation, validateSparkUiPrefix, cleanSparkUiPrefixForGrant } from './spark-ui-utils';
+
+/**
+ * Flex Jobs class
+ *
+ * Flex jobs supports Python and Scala language.
+ * The flexible execution class is appropriate for non-urgent jobs such as
+ * pre-production jobs, testing, and one-time data loads.
+ * Flexible job runs are supported for jobs using AWS Glue version 3.0 or later and G.1X or
+ * G.2X worker types but will default to the latest version of Glue (currently Glue 3.0.)
+ *
+ * Similar to ETL, we’ll enable these features: —enable-metrics, —enable-spark-ui,
+ * —enable-continuous-cloudwatch-log
+ *
+ */
+
+export interface PySparkFlexEtlJobProps extends JobProperties {
+
+  /**
```

Comment:
Are these in the RFC? Cannot see them

Reply:
There are the Props interface.. not required to be passed during invocation